docs/Makefile

```diff
@@ -8,6 +8,9 @@ BUILDDIR      = .
 
 install:
 	pip3 install sphinx sphinx-rtd-theme sphinx-autodoc-typehints ghp-import
+	rm -rf source/modules
+	mkdir -p source/modules
+	cd source/modules && git clone --branch st4000-exploration --depth 1 -- git@github.com:sublimelsp/LSP.git
```

Comment:
Can't we just ask the user to supply the path to the packages dir?

Reply:
I wanted it to be contained and work universally, also on CI. Why complicate it? What benefit would it have to use your own packages?